File: net/src/Sdl.Web.PublicContentApi/ContentModel/ContextData.cs
using System.Collections.Generic;

namespace Sdl.Web.PublicContentApi.ContentModel
{
    /// <summary>
    /// Context Data
    /// </summary>
    public class ContextData : IContextData
    {
        /// <summary>
        /// List of claim values to pass to query.
        /// </summary>
        public List<ClaimValue> ClaimValues { get; set; } = new List<ClaimValue>();
    }
}

File: java/pca-model/src/main/java/com/sdl/web/pca/client/contentmodel/ContextData.java
package com.sdl.web.pca.client.contentmodel;

import com.sdl.web.pca.client.contentmodel.generated.ClaimValue;

import java.util.ArrayList;
import java.util.Collection;
import java.util.Collections;
import java.util.List;

public class ContextData {
    private List<ClaimValue> claimValues = new ArrayList<>();

    public List<ClaimValue> getClaimValues() {
        return Collections.unmodifiableList(claimValues);
    }

    public void addClaimValule(ClaimValue value) {
        claimValues.add(value);
    }

    public void addClaimValues(ContextData values) {
        if (values != null && values.getClaimValues() != null) {
            claimValues.addAll(values.getClaimValues());
        }
    }

    public void addClaimValues(Collection<ClaimValue> values) {
        if (values != null) {
            claimValues.addAll(values);
        }
    }
}
